openshift-start-configured-cluster

```diff
@@ -43,6 +43,17 @@ for i in openshift-apiserver openshift-controller-manager; do
     sed -i 's|webhookTokenAuthenticators: null|webhookTokenAuthenticators:\n  - configFile: "webhook.yaml"|' $OC_CONFIG/$i/master-config.yaml
 done
 
+echo "Generating Keycloak client cert"
```

Comment:
you can probably just use oc adm ca create-master-certs

Reply:
Thanks for the hint! To be honest I totally missed that tool. I will give it a go.